shaping/output.go

```diff
@@ -86,6 +86,9 @@ func (b Bounds) LineHeight() fixed.Int26_6 {
 type Output struct {
 	// Advance is the distance the Dot has advanced.
 	Advance fixed.Int26_6
+	// Size is the scaling factor applied to this text during shaping, copied from
```

Comment:
Is it just me that is confused by this documentation?
I went to the Input struct to see what it means and I do understand it now, but to state that size is a scaling factor alone seems potentially confusing.

Reply:
I've updated it to simply state that the field is copied from `shaping.Input.Size`. That way we only need to document what it _means_ in one place.